delivery/components/TelemetryFeatures.yaml

```diff
@@ -5,4 +5,22 @@ TelemetryFeatures:
   properties:
     decisioningMethod:
       $ref: "./DecisioningMethod.yaml#/DecisioningMethod"
-      description: Decisioning Method
+      description: Decisioning Method used
+    executeMboxCount:
+      type: integer
+      format: int64
```

Comment:
Let's remove the format property for al the integer types in this particular file.  Since they are counts they won't get that high, so the default int type should work fine.